src/App.js

```diff
@@ -45,52 +87,47 @@ class Image extends React.Component {
                     return response.json();
                 })
                 .then((arrayOfData) => {
-                    this.setState(prevState => ({
-                        images: prevState.images.concat(arrayOfData),
-                        isFetching: false
-                    }));
+                    this.setState((prevState) => {
+                        const buffArray = prevState.images.concat(arrayOfData)
```

Comment:
Minor. I believe it would be more readable to write this like so:
```suggestion
                        const images = [...prevState.images, ...arrayOfData];
                        return {
                            count: images.length,
                            images,
                            isFetching: false
                        }
```